.github/FUNDING.yml

```diff
@@ -0,0 +1,4 @@
+# These are supported funding model platforms
+
+custom: ['https://amethyst.rs/donate']# Replace with up to 4 custom sponsorship URLs e.g., ['link1', 'link2']
```

Comment:
Please could we use either the vertical list format, or at least introduce a space after `]` 👀?
The YAML's valid and does what's intended, but the comment `#` being right next to the `]` made me wonder if it was.

Reply:
How about we just remove comments? Updated the PR